packages/client/e2e/pages/FxNewRfqPageObject.ts

```diff
@@ -0,0 +1,31 @@
+import { Page, TestInfo } from "@playwright/test"
+
+export class FxNewRfqPageObject {
+  constructor(
+    readonly page: Page,
+    readonly testInfo: TestInfo,
+  ) {}
+
+  async clickCurrency(currency: string) {
+    return await this.page
+      .getByTestId("tab-bar-tabs")
```

Comment:
this is where a Component would be useful, to tie in with reused UI elements - ref our conversation about the [RTC-QA-BUG-VERSION branch](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/compare/master...RTC-QA-BUG-VERSION)

Reply:
defer adding Components until later